node/src/components/storage.rs

```diff
@@ -370,6 +371,9 @@ pub struct Storage {
     enable_mem_deduplication: bool,
     /// Pool of loaded items.
     deploy_cache: BlobCache<<Deploy as Item>::Id>,
+    /// A map of era ID to a block height.
+    #[data_size(skip)]
```

Comment:
We can remove this if we get rid of the `RefCell`.

Reply:
Done